spec/controllers/facility_journals_controller_spec.rb

```diff
@@ -116,8 +116,9 @@ def create_order_details
           assigns[:journal].created_by.should == @admin.id
         end
 
-        it 'should have an is_successful value of true' do
-          assigns[:journal].is_successful? == true
+        it "has an is_successful value of true" do
+          expect(assigns[:journal].is_successful?).to be true
+          expect(assigns[:journal]).to be_successful
```

Comment:
It is. I wanted to have both tests until we completely phase out `is_successful`.